Augmentor/Operations.py

```diff
@@ -265,6 +265,154 @@ def do(image):
 
         return augmented_images
 
+class RandomBrightness(Operation):
+    """
+    This class is used to random change image brightness.
+    """
+    def __init__(self, probability, min_factor,max_factor):
+        """
+        required :attr:`probability` parameter
+
+        :func:`~Augmentor.Pipeline.Pipeline.random_brightness` function.
+
+        :param probability: Controls the probability that the operation is
+         performed when it is invoked in the pipeline.
+        :param min_factor: The value between 0.0 and max_factor 
+         that define the minimum adjustment of image brightness. 
+         The value  0.0 gives a black image,The value 1.0 gives the original image, value bigger than 1.0 gives more bright image.
+        :param max_factor: A value should be bigger than min_factor.
+         that define the maximum adjustment of image brightness.
+         The value  0.0 gives a black image, value 1.0 gives the original image, value bigger than 1.0 gives more bright image.
+        
+        :type probability: Float
+        :type max_factor: Float
+        :type max_factor: Float
+        """
+        Operation.__init__(self, probability)
+        self.min_factor = min_factor
+        self.max_factor = max_factor
+    def perform_operation(self, images):
+        """
+        Random change the passed image brightness.
+
+        :param images: The image to convert into monochrome.
+        :type images: List containing PIL.Image object(s).
+        :return: The transformed image(s) as a list of object(s) of type
+         PIL.Image.
+        """
+
+        def do(image):
+
+            factor = np.random.uniform(self.min_factor, self.max_factor)
```

Comment:
This would need to be moved to outside of the scope of `do()` to somewhere within `perform_operation()`.

Reply:
Thank you for  pointing out my mistake.  I have moved the 
factor = np.random.uniform(self.min_factor, self.max_factor)
out of function do()